gradle/publish-java.gradle

```diff
@@ -28,3 +28,53 @@ publishing {
     }
   }
 }
+
+gradle.taskGraph.whenReady({ graph ->
+  tasks.withType(Jar).each { jar ->
+    jar.doFirst {
+      def projectDependencies = []
+      def runtimeList = []
+
+      configurations.runtimeClasspath
+              .collect { it.name - ".jar" }
+              .each { dependency ->
+                if (dependency.startsWith("geode-")) {
+                  projectDependencies.add(dependency)
+                } else {
+                  runtimeList.add(dependency)
+                }
+              }
+
+      projectDependencies.clone().each { projectDependency ->
+        def geodeProject = projectDependency - "-${version}.jar"
+          if (projectDependencies.contains(geodeProject)) {
+            def parentProject = project(":$geodeProject" - "-$version")
+            if (parentProject != null) {
+              def collect = parentProject.configurations.runtimeClasspath.collect { it.name - ".jar" }
+              runtimeList.removeAll(collect)
+              projectDependencies.removeAll(collect)
+            }
+          }
+        }
+
+      manifest {
+        attributes(
+                "Manifest-Version": "1.0",
```

Comment:
Would love if we could merge the new attributes in, instead of overwriting. Any chance of that, before we merge to `develop` ?

Reply:
Yes.. we intended to merge this into `develop`. But we wanted to run against this feature branch first.

Reply:
We'll look into this before we submit PR to merge into `develop`